.gitignore

```diff
@@ -16,6 +16,9 @@ stage
 .pc/
 .cache/
 .mypy_cache/
+.pytest_cache/
```

Comment:
I see a `.pytest_cache/.gitignore` file with the contents:

```
# Created by pytest automatically.
*
```

so I don't know if this one is 100% required; no harm in having it though!